src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -365,7 +365,7 @@ class ValidatorStore extends BaseStore {
                 break;
               }
               iterations += 1;
-              if (await myAccountStore.checkEntityInAccount('cluster', 'validator_count', payload.get(CLUSTER_DATA).validatorCount)) {
+              if (await myAccountStore.checkEntityInAccount('cluster', 'validatorCount', payload.get(CLUSTER_DATA).validatorCount)) {
```

Comment:
@axelrod-blox Did you check this works? I think it will break. It looks like it takes the 'validator_count' property from the getter contract.

Reply:
checked it yesterday everything workking well